javascript/javascript2/week3/delayfunction/delayfunction.js

```diff
@@ -0,0 +1,9 @@
+// Create a function called runAfterDelay. It has two parameters: delay and callback. When called the function should wait delay seconds and then call the provided callback function. Try and call this function with different delays and different callback functions
+function runAfterDelay(delay, callback) {
+  setTimeout(() => {
+    console.log(callback);
```

Comment:
Just calling the function is sufficient...
```suggestion
    callback;
```